code/modules/mob/living/simple_animal/hostile/illusion.dm

```diff
@@ -42,6 +48,15 @@
 		return ..()
 
 
+/mob/living/simple_animal/hostile/illusion/AttackingTarget()
+	..()
+	if(istype(target, /mob/living) && prob(multiply_chance))
```

Comment:
Inb4 zombie greytide interaction leads to hordes of undead assistants that constantly reproduce by attacking dead corpses or something.